utils/ci/miri.sh

```diff
@@ -8,14 +8,15 @@ rustup component add miri
 cargo miri setup
 
 cargo miri test --no-default-features -- -Zmiri-seed=42 -- -Zunstable-options --exclude-should-panic
-cargo miri test --features=serde1,log -- -Zmiri-seed=42 -- -Zunstable-options --exclude-should-panic
+cargo miri test --features=log -- -Zmiri-seed=42 -- -Zunstable-options --exclude-should-panic
 cargo miri test --manifest-path rand_core/Cargo.toml
+cargo miri test --manifest-path rand_core/Cargo.toml --features=serde1
 cargo miri test --manifest-path rand_core/Cargo.toml --no-default-features
 #cargo miri test --manifest-path rand_distr/Cargo.toml # no unsafe and lots of slow tests
 cargo miri test --manifest-path rand_isaac/Cargo.toml --features=serde1
 cargo miri test --manifest-path rand_pcg/Cargo.toml --features=serde1
 cargo miri test --manifest-path rand_xorshift/Cargo.toml --features=serde1
-cargo miri test --manifest-path rand_xoshiro/Cargo.toml
+cargo miri test --manifest-path rand_xoshiro/Cargo.toml --features=serde1
```

Comment:
Why not test checkpointing for Xoshiro?

Reply:
This PR enables the serialization tests for Xoshiro. They were not enabled before!